flask_datadog.py

```diff
@@ -169,12 +172,21 @@ def after_request(self, response):
         if self.config['DATADOG_RESPONSE_AUTO_TAG']:
             self.add_request_tags(['status_code:%s' % (response.status_code, )])
 
-        # Emit our timing metric
-        self.statsd.timing(self.config['DATADOG_RESPONSE_METRIC_NAME'],
+        tags = self.get_request_tags()
+
+        # Emit timing metric
+        self.statsd.timing(self.config['DATADOG_RESPONSE_TIME_METRIC_NAME'],
                            elapsed,
-                           self.get_request_tags(),
+                           tags,
                            self.config['DATADOG_RESPONSE_SAMPLE_RATE'])
 
+        # Emit response size metric
+        size = len(response.data)
```

Comment:
I used `len(response.data)` because of the bug in jsonify: https://github.com/pallets/flask/issues/1877.
`content-length` should work best for the size metric. I'll add workaround for jsonify content-length issue in my view functions.